packages/page-accounts/src/modals/Delegate.tsx

```diff
@@ -3,11 +3,11 @@
 
 import { AmountValidateState } from '../Accounts/types';
 
+import { Conviction } from '@polkadot/types/interfaces';
```

Comment:
These are all types, so I guess it is a special case again for `@polkadot/types/interfaces*` (They are wildly inconsistent in all the repos, some places they are treated correctly as types, other places not)

Reply:
This one is weird though, since I don' know why it jumped to the top of this section and not the previous section rather.

Reply:
It actually jumped into the types section - but as I said, I did not touched the group separation configuration, because it would cause hundreds of files to change. The breaks between the groups are indifferent to linter configuration and it does not change them. However, I will manually adjust them in a minute in the places you've pointed